core/include/gnuradio-4.0/BlockModel.hpp

```diff
@@ -495,6 +498,9 @@ class BlockWrapper : public BlockModel {
 
     UICategory uiCategory() const override { return T::DrawableControl::kCategory; }
 
+    property_map uiConstraints() const override { return blockRef().ui_constraints; }
```

Comment:
I don't think that this needs/warrants an extra accessor since the data is already available via the existing settings interface.